rotkehlchen/tests/unit/decoders/test_aura_finance.py

```diff
@@ -0,0 +1,289 @@
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(FVal(gas_str)),
+            location_label=user_address,
+            notes=f'Burn {gas_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=18,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=Asset('eip155:8453/erc20:0x4158734D47Fc9692176B5085E0F52ee0Da5d47F1'),
+            balance=Balance(FVal(claim_amt_1)),
+            location_label=user_address,
+            notes=f'Claim {claim_amt_1} BAL from Aura Finance',
+            address=string_to_evm_address('0xEe374580BFf150be6b955954aC3b9899D890cB57'),
+            counterparty=CPT_AURA_FINANCE,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=19,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.REWARD,
+            asset=Asset('eip155:8453/erc20:0x1509706a6c66CA549ff0cB464de88231DDBe213B'),
+            balance=Balance(FVal(claim_amt_2)),
+            location_label=user_address,
+            notes=f'Claim {claim_amt_2} AURA from Aura Finance',
+            counterparty=CPT_AURA_FINANCE,
+            address=string_to_evm_address('0x8b2970c237656d3895588B99a8bFe977D5618201'),
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('base_accounts', [['0x19e4057A38a730be37c4DA690b103267AAE1d75d']])
+def test_aura_finance_lock_aura_from_base_to_ethereum(base_inquirer, base_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xe0a6fd1bd40451d4b42c520b41a39ab569bf4aae43b741efbe228da40fed91ad')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=tx_hash)
+    user_address, timestamp, gas_str, bridge_fee_amt, locked_amt = base_accounts[0], TimestampMS(1732631175000), '0.000007432156846576', '0.011914017676630994', '90'  # noqa: E501
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(FVal(gas_str)),
+            location_label=user_address,
+            notes=f'Burn {gas_str} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
```

Comment:
Shouldn't it be a BRIDGE/FEE ?

Reply:
there's no BRIDGE/FEE. `BRIDGE` only exists as `HistoryEventSubType`